brambling/models.py

```diff
@@ -299,6 +302,30 @@ def create_defaults(app_config, **kwargs):
             ])
 
 
+class OrganizationMember(models.Model):
+    EDIT = '1-edit'
+    VIEW = '2-view'
+    OWNER = '0-owner'
```

Comment:
This may just be a convention I'm not aware of, but what are these numbers, here?